### PR TITLE
Automate Lighthouse Testing

### LINKED ISSUE #5255: Add Lighthouse audit to compare live to dev/multidev
Re: [pantheonio PR 662](https://github.com/pantheon-systems/pantheonio/pull/662)

Priority: High 

## Issue Description
Add Lighthouse benchmarking to automated testing to facilitate performance benchmarking for changes to the docs site.

## Suggested Resolution
Replicate & modify process created by @ataylorme for pantheonio as needed to implement for the docs site.

- Uses NodeJS to run Lighthouse against the dev/multidev environment and pantheon.io/docs.
- Jest is used to test the resulting scores.
- Posts the HTML reports back to the GitHub PR

### PR DESCRIPTION
**Pull request by alexfornuto**
Closes #5255 

## Effect
PR includes the following changes:
- Refactors deployment code to better accept new tests with sub-linear increase in complexity.
- Implement lighthouse testing as part of CI.

## Remaining Work
- [x] Refactor existing CI scripts to better abstract commonly used code.
- [x] Install and configure Lighthouse testing in CircleCi, which will report to GitHub.
- [x] Technical review
- [x] Copy review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `84` | `82` | `-2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `88` | `80` | `-8` |
| Accessibility Score | `95` | `95` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16309/artifacts/0/artifacts/lighthouse_data/lighthouse-audit-reference.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16309/artifacts/0/artifacts/lighthouse_data/lighthouse-audit-dev.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `74` | `72` | `-2` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `90` | `80` | `-10` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16313/artifacts/0/artifacts/lighthouse_data/lighthouse-audit-reference.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16313/artifacts/0/artifacts/lighthouse_data/lighthouse-audit-dev.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `90` | `80` | `-10` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16317/artifacts/0/artifacts/lighthouse_data/lighthouse-audit-reference.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16317/artifacts/0/artifacts/lighthouse_data/lighthouse-audit-dev.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `90` | `80` | `-10` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16321/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16321/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `90` | `90` | `0` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16329/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16329/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `90` | `90` | `0` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16333/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16333/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `90` | `90` | `0` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16337/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16337/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `90` | `90` | `0` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16345/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16345/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `89` | `100` | `11` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16349/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16349/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `89` | `100` | `11` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16356/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16356/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by alexfornuto** [comment]
<span style=“color:green;”> text goes here</span>

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `89` | `100` | `11` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16363/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16363/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by alexfornuto** [comment]
```diff
+just a test
-ignore me
```

**Comment by alexfornuto** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `89` | `100` | <pre class="highlight highlight-source-diff">11</pre> |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16363/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16363/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)



class="highlight highlight-source-diff"

File: gatsby/src/layout/changelog-preview/index.js [hunk 1]
```diff
@@ -17,10 +17,10 @@ const ChangelogPreview = (props) => {
 						<MDXRenderer>{current.node.fields.markdownBody.childMdx.body}</MDXRenderer>
 					</MDXProvider>
 					<Link
-						to={url}
+						to={`${url}/`}
 						className="cta"
 					>
-						More
+						Read More
```
Comment (alexfornuto) [inline gatsby/src/layout/changelog-preview/index.js]: Highlighting as a change that requires Copy Review

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `89` | `100` | `11` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16375/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16375/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `0` | `-100` |
| Best Practice Score | `93` | `0` | `-93` |
| SEO Score | `89` | `0` | `-89` |
| Accessibility Score | `88` | `0` | `-88` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16383/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16383/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `0` | `-100` |
| Best Practice Score | `93` | `0` | `-93` |
| SEO Score | `89` | `0` | `-89` |
| Accessibility Score | `88` | `0` | `-88` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16397/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16397/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `89` | `100` | `11` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16402/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16402/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `89` | `100` | `11` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16465/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16465/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `0` | `-100` |
| Best Practice Score | `93` | `0` | `-93` |
| SEO Score | `89` | `0` | `-89` |
| Accessibility Score | `88` | `0` | `-88` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16479/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16479/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `89` | `100` | `11` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16485/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16485/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `87` | `96` | `9` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16493/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16493/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `87` | `96` | `9` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16497/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16497/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `87` | `96` | `9` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16502/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16502/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `100` | `100` | `0` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `87` | `96` | `9` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16506/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16506/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `86` | `86` | `0` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `87` | `96` | `9` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16518/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16518/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: gatsby/src/layout/changelog-preview/index.js (hunk 1 above)
Comment (alexfornuto) [reply gatsby/src/layout/changelog-preview/index.js]: @carolyn-shannon approved during review.

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `84` | `79` | `-5` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `88` | `88` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16606/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16606/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `75` | `88` | `13` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `88` | `88` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16607/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16607/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `84` | `87` | `3` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `88` | `88` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16611/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16611/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by populist** [comment]
+1 from me. Nice to see us leveraging Lighthouse for the already rocking docs site!

**Comment by sugaroverflow** [comment]
+1 from me too - thanks for giving me an overview!

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `83` | `0` | `-83` |
| Best Practice Score | `93` | `0` | `-93` |
| SEO Score | `88` | `0` | `-88` |
| Accessibility Score | `96` | `0` | `-96` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16624/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16624/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `82` | `73` | `-9` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `88` | `88` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16628/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16628/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5320 | Difference |
| --- | --- | --- | --- |
| Performance Score | `83` | `87` | `4` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `88` | `88` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5320` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16629/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/16629/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Pull request merged**